src/components/layout/Undernames/Undernames.tsx

```diff
@@ -476,7 +561,11 @@ function Undernames() {
                       setValue={(e) => {
                         e ? setTTL(+e) : setTTL(undefined);
                       }}
-                      validationPredicates={{}}
+                      validityCallback={(isValid) => setChangesValid(isValid)}
+                      validationPredicates={{
+                        [VALIDATION_INPUT_TYPES.VALID_TTL]: (ttlSeconds) =>
+                          arweaveDataProvider.validateTTLSeconds(+ttlSeconds),
```

Comment:
this is just a util method, it doesn't require an API call so no need to attach to this provider. move it to a separate util function